### ISSUE TITLE
Create speech_to_point() function (03/14/18)

### ISSUE THREAD

**Issue by dmarulli** [Streets-Data-Collaborative/geo-street-talk-global#8]
Great idea @charlie-moffett ! To make this work, you'd probably need to reverse the function mapping and take a human statement like "between Broadway and third" and return a lat, long pair

From @patwater in the webinar :) 

CC @vr00n

**Comment by vr00n** [comment]
Yep, this application would fall squarely into the Autonomous Vehicle use-case where a core component of `In-vehicle interaction` is to be able to communicate destination information to the vehicle in a naturalistic manner without needing to interact with a phone.

While Alexa is the mainstream speech-to-text integration, I'd actually think that doing this with Google could allow us to reach out to Waymo folks.

**Comment by dmarulli** [comment]
Create a speech_to_point() function that receives a street_name, a street_from, and a street_to; and returns a lat/lng pair--presumably using inverse_streetTalk()
